locale/en/messages.json

```diff
@@ -666,23 +666,28 @@
   "whm.assize.checklist.requirement.assize.name": "<0/> uses",
   "whm.benison.suggestion.content": "Use <0/> more frequently. Frequent uses can mitigate a large amount of damage over the course of a fight, potentially resulting in less required healing GCDs.",
   "whm.benison.suggestion.why": "About {_usesMissed} uses of <0/> were missed by holding it for at least a total of {0}.",
-  "whm.dots.requirement.uptime-a2.name": "<0/> uptime",
-  "whm.dots.requirement.uptime-a3.name": "<0/> uptime",
-  "whm.dots.rule.description": "As a White Mage, DoTs are significant portion of your sustained damage. Aim to keep them up at all times.",
+  "whm.dots.requirement.uptime-dia.name": "<0/> uptime",
+  "whm.dots.rule.description": "As a White Mage, Dia is significant portion of your sustained damage. Aim to keep it up at all times.",
   "whm.dots.rule.name": "Keep your DoTs up",
-  "whm.dots.suggestion.clip-a2.content": "Avoid refreshing DoTs significantly before their expiration, this will allow you to cast more Stone IV. (Note: We do not yet consider using Aero II for initial damage on the move)",
-  "whm.dots.suggestion.clip-a2.why": "{0} of {1} lost to early refreshes.",
-  "whm.dots.suggestion.clip-a3.content": "Avoid refreshing DoTs significantly before their expiration, this will allow you to cast more Stone IV.",
-  "whm.dots.suggestion.clip-a3.why": "{0} of {1} lost to early refreshes.",
+  "whm.dots.suggestion.clip-dia.content": "Avoid refreshing Dia significantly before their expiration, this will allow you to cast more Glare.",
+  "whm.dots.suggestion.clip-dia.why": "{0} of {1} lost to early refreshes.",
+  "whm.lily-cap.requirement.name": "Avoid capping your Lilies",
+  "whm.lily-cap.suggestion.content": "Use <0/> or <1/> regularly to avoid capping your lilies",
+  "whm.lily-cap.suggestion.why": "{0, plural, one {# lily} other {# lilies}} went unused.",
+  "whm.lily-blood.checklist.name": "Spend your Blood Lily",
+  "whm.lily-blood.checklist.content": "Use <0/> when it is available. It is a major source of your DPS.",
+  "whm.lily-blood.requirement.name": "Blood Lilies Generated",
+  "whm.lily-blood.suggestion.content": "Use <0/> to avoid wasting blood lily growth.",
+  "whm.lily-blood.suggestion.why": "{0, plural, one {# blood lily} other {# blood lilies}} did not bloom due to early lily use.",
   "whm.overheal.requirement.all": "Overheal (all sources)",
   "whm.overheal.requirement.hot": "Overheal (HoT)",
   "whm.overheal.requirement.nonhot": "Overheal (non-HoT)",
   "whm.overheal.rule.description": "Avoid wasting heals by healing for more than required to fill a target's HP bar. While some overheal is inevitable, overheal only serves to generate more enmity, for no gain. Being efficient with healing additionally helps with your MP management.",
-  "whm.overheal.rule.name": "Avoid overheal",
+	"whm.overheal.rule.name": "Avoid overheal",
```

Comment:
Hm... I have _no_ idea why this is indenting differently. Weird. Not an issue I guess, the build will automatically override this anyway.